### ISSUE TITLE
Plugin not showing up in Tasker

### ISSUE THREAD

**Issue by Anon0ne** [MarkAdamson/home-assistant-plugin-for-tasker#18]
Hello, I am running a Pixel 3XL on Android 10. Tasker is updated to latest, and your plugin is also updated to the latest version. However, after installation and even trying to run the app and restarting Tasker multiple times (including phone reboots) I cannot get this plugin to show up in the list under Plugins or Add-ons.

**Comment by MarkAdamson** [comment]
Hi @Anon0ne, please can you just confirm that you are searching under 'Action' plugins, and not 'Event' or 'State' plugins?

Thank you.

**Comment by sebastian-ruiz** [comment]
I have the same problem. 

I am on the 'Tasks' tab in the home screen in Tasker. I add a new task, and in the new task I add an action. It then gives me a list of actions. Under 3rd Party Action, there is no Home Assistant plugin. When I filter the actions by searching for 'home' the Home Assistant plugin also does not appear.

I don't know what you mean by 'Action', 'Event' or 'State' plugins.

**Comment by Anon0ne** [comment]
> I have the same problem.
> 
> I am on the 'Tasks' tab in the home screen in Tasker. I add a new task, and in the new task I add an action. It then gives me a list of actions. Under 3rd Party Action, there is no Home Assistant plugin. When I filter the actions by searching for 'home' the Home Assistant plugin also does not appear.
> 
> I don't know what you mean by 'Action', 'Event' or 'State' plugins.

Hey, this issue ended up solving itself after an update. Not sure why, but it began showing up later. I would try to reset phone first, otherwise give it a few days maybe for a Tasker update.

**Comment by Anon0ne** [comment]
They do need to look into the plugin isn't showing up as soon as the app opens though, not sure.

**Comment by sebastian-ruiz** [comment]
I found it! 

When it asks 'Select Action Category' there is a category called 'Plugin' and this is where the 'Home Assistant plugin' is in. Not in '3rd Party'.

Thanks for your help!

**Comment by codingcatgirl** [comment]
Nope, it's not under Plugins either. Pixel 3a here as well. And yes, i'm trying to add an action.

**Comment by codingcatgirl** [comment]
Update: Restart your phone and it shows up.
